docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -34,12 +38,24 @@ variance_pipeline = pipeline(
     ]
 )
 ```
-</details>
 
-You can use `describe` to discover what nodes are part of the pipeline:
+Kedro determines the order of execution of these nodes based on the inputs and outputs specified for each node. In this example:
+1. The first node computes the length of `xs` and outputs it as `n`.
+2. The second node calculates the mean using `xs` and `n`, and outputs it as `m`.
+3. The third node computes the mean sum of squares (mean_sos) using `xs` and `n`, and outputs it as `m2`.
+4. The fourth node calculates the variance using the mean (`m`) and mean sum of squares (`m2`), and outputs it as `v`.
 
-<details>
-<summary><b>Click to expand</b></summary>
+Kedro's dependency resolution ensures that each node runs only after its required inputs are available from the outputs of previous nodes. This way, the nodes are executed in the correct order automatically, based on the defined dependencies.
```

Comment:
Maybe it helps to print `pipline.nodes`  which is toposorted already?

Reply:
It's the section below titled "How to receive information about the nodes in a pipeline" that shows that output, so I don't think it's a good idea to duplicate it on the same page.